### PR TITLE
[FSSDK-9630] Fix: Code examples

### PR DESCRIPTION
**Pull request by localheinz**
## Summary

- fixes code examples

## Test plan

n/a

## Issues

n/a

[FSSDK-9630]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,23 +37,41 @@ php composer.phar require optimizely/optimizely-sdk
 Create the Optimizely client, for example:
 
 ```php
+<?php
```
Comment (localheinz) [inline README.md:40]: Adding opening tags here allows the fenced code blocks to be properly interpreted as PHP code by IDEs, for example, by [PhpStorm](https://www.jetbrains.com/phpstorm/).

File: README.md [hunk 2]
```diff
@@ -37,23 +37,41 @@ php composer.phar require optimizely/optimizely-sdk
...
+
+use Optimizely\Optimizely;
```
Comment (localheinz) [inline README.md:42]: Adding imports here allows code-completion in fenced code blocks by IDEs, for example, by [PhpStorm](https://www.jetbrains.com/phpstorm/).

**Comment by mikechu-optimizely** [comment]
> Before I can do that, though, please read and follow the [CONTRIBUTING.md](https://github.com/optimizely/php-sdk/blob/master/CONTRIBUTING.md). Once you've filled out the CLA, ping me here please 👍

I do not currently see your name in our CLA signed list @localheinz.  Definitely, fill out the [CLA form](https://docs.google.com/a/optimizely.com/forms/d/e/1FAIpQLSf9cbouWptIpMgukAKZZOIAhafvjFCV8hS00XJLWQnWDFtwtA/viewform) when you get a moment.  I'm not allowed to use any open-source contributions without the submission…. Uggg 😩 Legal.

**Comment by localheinz** [comment]
@mikechu-optimizely 

Done!

**Comment by mikechu-optimizely** [comment]
> @mikechu-optimizely
> 
> Would you like me to apply the changes from all of my pull requests here into a single pull request?

That'd be super helpful.  Yes, please.  I can pull them off your fork much more completely that way.  

Thanks and cheers.

**Comment by localheinz** [comment]
@mikechu-optimizely 

I have cherry-picked the commits from #274, #275, and #276 into the branch for this pull request.

**Comment by mikechu-optimizely** [comment]
Besten Dank, Herr!  

I created our internal ticket (FSSDK-9630) to get this into the main branch.

**Comment by mikechu-optimizely** [comment]
We'll be pointing this tomorrow and then rolling it into the next sprint (ahhh ceremonies and planning).

**Comment by mikechu-optimizely** [comment]
Point given (thanks for shortening that for us). Moved into planning for the next sprint (starts next week)

**Comment by mikechu-optimizely** [comment]
Yikes. This slipped through our planning.  I just noticed it when preparing to cut a patch version.  

Review looks good. I need to bring a copy into our org so our back-end CI tests run.

**Review by mikechu-optimizely — APPROVED** [review]
LGTM

**Comment by mikechu-optimizely** [comment]
@localheinz Decided not to `sort-packages` and remove the ReturnTypeWillChange decorators?

**Comment by mikechu-optimizely** [comment]
Thanks. Pulling this into the org so the CI runs....

**Comment by mikechu-optimizely** [comment]
I'm closing this PR in exchange for #274

**Pull request closed without merging**